src/test/java/org/opensearch/jobscheduler/transport/GetJobTypeTransportActionTests.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.jobscheduler.transport;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.opensearch.action.ActionListener;
+import org.opensearch.action.support.ActionFilters;
+import org.opensearch.tasks.Task;
+import org.opensearch.test.OpenSearchTestCase;
+import org.opensearch.transport.TransportService;
+
+import static org.mockito.Mockito.mock;
+
+public class GetJobTypeTransportActionTests extends OpenSearchTestCase {
+
+    private GetJobTypeTransportAction action;
+
+    private Task task;
+
+    private GetJobTypeRequest request;
+
+    private ActionListener<GetJobDetailsResponse> response;
+
+    @Before
+    public void setUp() throws Exception {
+        super.setUp();
+        action = new GetJobTypeTransportAction("", mock(TransportService.class), mock(ActionFilters.class));
+        request = new GetJobTypeRequest("demo_job_type", "extension_id");
+        task = mock(Task.class);
+        response = new ActionListener<GetJobDetailsResponse>() {
+            @Override
+            public void onResponse(GetJobDetailsResponse jobDetailsResponse) {
+                // onResponse will not be called as we do not have the AD index
+                Assert.assertEquals("success", jobDetailsResponse.getResponse());
+            }
+
+            @Override
+            public void onFailure(Exception e) {
+                // Assert.assertTrue(true);
+            }
+        };
+
+    }
+
+    @Test
+    public void testGetJobTypeTransportAction() {
+        action.doExecute(task, request, response);
```

Comment:
Should there be an assertion here?  Expected exception perhaps?

Reply:
No, it is a void method, the assertion is in listener.

Reply:
You can still do an [`assertDoesNotThrow()`](https://junit.org/junit5/docs/5.8.2/api/org.junit.jupiter.api/org/junit/jupiter/api/Assertions.html#assertDoesNotThrow(org.junit.jupiter.api.function.Executable)).

Reply:
Junit 5 is not supported in JS